sdks/python/apache_beam/coders/coders_test_common.py

```diff
@@ -143,9 +143,9 @@ def test_bytes_coder(self):
 
```

Comment:
I haven't done any work for making the tests pass in Python 3. There are many errors in coders due to string compatibility issues though. If we can get a version which builds on Python 3 up to a remote branch, it would be easy to communicate progress. I saw that you created a tox env in a PR.